### PR TITLE
[NTR] Pax6 GABAergic cortical interneuron

### LINKED ISSUE #2595: [NTR] Pax6 GABAergic cortical interneuron

### PR DESCRIPTION
**Pull request by AvolaAmg**
Fixes #2595

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32517,6 +32519,18 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
...
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4042032 "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses PAX6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters PAX6.")
+AnnotationAssertion(terms:contributor obo:CL_4042032 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042032 "2024-10-11T13:25:51Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042032 "PMID:37824655")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042032 "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/")
+AnnotationAssertion(rdfs:label obo:CL_4042032 "PAX6 GABAergic cortical interneuron"@en)
+SubClassOf(obo:CL_4042032 obo:CL_4023064)
+SubClassOf(obo:CL_4042032 ObjectSomeValuesFrom(obo:RO_0002202 obo:UBERON_0004026))
+SubClassOf(obo:CL_4042032 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000012318))
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32532]: Is PAX6 translated? PR terms are proteins, so we need to make sure that aside of transcriptomic data we also have protein data (mass spec, flow cytometry, abs...)

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32517,6 +32519,18 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
 SubClassOf(obo:CL_4042027 obo:CL_0002614)
 SubClassOf(obo:CL_4042027 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001903))
 
+# Class: obo:CL_4042032 (PAX6 GABAergic cortical interneuron)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4042032 "A transcriptomically distinct GABAergic neuron located in the cerebral cortex that expresses PAX6. The standard transcriptomic reference data for this cell type can be found on the CellxGene census under the collection: 'Transcriptomic cytoarchitecture reveals principles of human neocortex organization', dataset: 'Supercluster: CGE-derived interneurons', Author Categories: 'CrossArea_subclass', clusters PAX6.")
+AnnotationAssertion(terms:contributor obo:CL_4042032 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042032 "2024-10-11T13:25:51Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042032 "PMID:37824655")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32527]: Is this reference meant to annotate the definition?

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -32517,6 +32519,18 @@ AnnotationAssertion(rdfs:label obo:CL_4042027 "GABAergic interneuron of the post
...
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4042032 "https://cellxgene.cziscience.com/e/e4ddac12-f48f-4455-8e8d-c2a48a683437.cxg/")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:32528]: Is this reference meant to annotate the definition?

**Review by aleixpuigb — DISMISSED** [review]
Please find inline comments.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32532]: it is a neuronal type but I have not found proteomics data for it. I can have an extra look

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32527]: It is meant to annotate the definition by looking at the scRNAseq data

File: src/ontology/cl-edit.owl (hunk 3 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32528]: yes will have a look at that too

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32532]: Cannot find any proteomics data, however this neuronal type is present in CellxGenes datasets. should I change the definition with "expresses the transcript PAX6"?

Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:32532]: No, the definition is fine, but if there is no evidence than the transcript is being translated, we shouldn't use PR. Maybe use [entrez:5080](https://www.ncbi.nlm.nih.gov/gene/5080)? @dosumis what do you think?

Comment (dosumis) [reply src/ontology/cl-edit.owl:32532]: We've gone back and forth on this. Infrastructure issues have prevented us moving to using gene IDs.  Many other neuronal cell types have the same issue.  Not ideal, bit let's just get this out.  We can add a comment to clarify evidence.

**Comment by AvolaAmg** [comment]
#gogoedit

**Review by dosumis — APPROVED** [review]
Approved pending update to PRO import

**Pull request merged**